### PR TITLE
Start renaming and cleaning

### PR DESCRIPTION
**Pull request by VinceMacBuche**
This is a WIP and example, i need to add more tests on edge cases ...

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala [hunk 1]
```diff
@@ -242,6 +242,11 @@ class TestNodeAndGlobalParameterLookup extends Specification {
       test(all(_), """${something . cfengine}""", List(NonRudderVar("${something . cfengine}")))
     }
 
+    "parse blank test" in {
+      test(all(_), "      ",
+        List(CharSeq("      ")))
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala:247]: you still need empty to parse ""
